en/Obsidian/Community code of conduct.md

```diff
@@ -2,24 +2,40 @@
 - **Belligerence**: deliberately attempting to make others feel bad, name-calling, singling out others for derision or exclusion. For example, telling someone they’re not a real programmer or that they don’t belong.
-- **Being especially unpleasant**: for example, if we’ve received reports from multiple members of annoying or rude behavior.
+- **Being especially unpleasant**: for example, if we’ve received reports from multiple members of annoying or rude behavior. ^432ea1
 
-#### Other behaviors
+### Other behaviors
+
+We strongly discourage these behaviors. Please don’t do them. These behaviors do not result in a ban or warning, but they will greatly increase one’s chance of [[#^432ea1|being especially unpleasant]].
 
 - **Sea-lioning**: continuously pursuing an agenda by questioning and provoking other members. Individual provocations and questions are often perfectly acceptable, but in the case of sea-lioning, they accumulate to unhealthy levels, creating a hostile and negative conversation.
 - **"Feed me"**: asking others to solve problems for you without putting in any effort yourself. This is particularly egregious when the problem-solving is non-trivial and yet the user begs for someone else to do the work for them.
 - **Campaigning**: continuously making the same requests. This applies to both feature requests and to requests for help. Do not post the same question in multiple places in short timeframes, and do not ask developers for the same features or bug fixes repeatedly.
 - **Parasitism**: self-promoting of yours or others' initiatives without contributing to the community otherwise. We encourage self-promotion (it is awesome to see what Obsidian users are up to) but will not tolerate users whose only purpose in the community is to post links or solicitations to initiatives outside the community.
 
+## Encouraged behaviors
+
+Over the years of running our community, we’ve noticed some behaviors that positively contribute to the community.
+
+If you’re looking to improve your communication skills as well as becoming a better community citizen, you can:
+
+- **Try before asking**: if you’re asking for help, please first try searching the help docs and Discord/forum history to see if the question has always been asked. When asking the question, sharing what you have tried but doesn’t work would help others help you.
+- **Keep an open mind**: because of how customizable Obsidian can be, some people might have a completely different set-up or workflow from you. Do not judge others for their choices.
+- **Step away when needed**: the community is a place for us to grow together, not a debate competitions. There are no arguments that you have to win. If a discussion turns into an argument and escalates, try your best to step away and be the bigger person.
```

Comment:
> [...] try your best to step away. 

Cut the "be the bigger person" part as it suggests a superiority/piousness competition.